src/main/java/chess/domain/Pieces.java

```diff
@@ -0,0 +1,126 @@
+package chess.domain;
+
+import java.math.BigInteger;
+import java.util.ArrayDeque;
+import java.util.ArrayList;
+import java.util.Deque;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public final class Pieces {
+
+    public static final char FIRST_RANK = 'a';
+    public static final char LAST_RANK = 'h';
+    private static final char MIDDLE_RANK = 'd';
+    public static final int FIRST_FILE_OF_WHITE = 0;
+    private static final int LAST_FILE_OF_WHITE = 1;
+    public static final int FIRST_FILE_OF_BLACK = 7;
+    private static final int LAST_FILE_OF_BLACK = 6;
+    private static final char QUEEN_DEFAULT_RANK_POSITION = 'd';
+    private static final char KING_DEFAULT_RANK_POSITION = 'e';
+
+    private final List<Piece> pieces;
+
+    private Pieces(final List<Piece> pieces) {
+        this.pieces = pieces;
+    }
+
+    public static Pieces createBlackPieces(Pieces whitePieces) {
+        List<Piece> pawns = getBlackPawns(whitePieces);
+        List<Piece> piecesWithoutPawns = getPiecesWithoutPawn(whitePieces);
+        return getBlackPieces(pawns, piecesWithoutPawns);
+    }
+
+    private static List<Piece> getBlackPawns(Pieces whitePieces) {
+        return whitePieces.pieces.stream()
+                .filter(piece -> piece.isSameShape(Shape.PAWN))
+                .map(piece -> piece.getNewPiece(LAST_FILE_OF_BLACK))
+                .collect(Collectors.toList());
+    }
+
+    private static List<Piece> getPiecesWithoutPawn(Pieces whitePieces) {
+        return whitePieces.pieces.stream()
+                .filter(piece -> !piece.isSameShape(Shape.PAWN))
```

Comment:
Not은 앞에 `!`를 붙이게 되는데, 이렇게 되면 가독성이 떨어지기도 해서
반대의 메서드도 하나 만들어주기도 합니다. `isDifferentShape`

이 부분은 취향 차이가 있을 수 있으니 참고만 해주세요~

```suggestion
                .filter(piece -> piece.isDifferentShape(Shape.PAWN))
```